# Rover x,y positon on the map
int8 x
int8 y

# Rover orientation
int8 ORIENTATION_NORTH = 0
int8 ORIENTATION_EAST = 1
int8 ORIENTATION_SOUTH = 2
int8 ORIENTATION_WEST = 3
int8 orientation